packages/skill-lesson/path-to-purchase/pricing.tsx

```diff
@@ -624,167 +624,181 @@ export const Pricing: React.FC<React.PropsWithChildren<PricingProps>> = ({
-                </div>
-              )}
-            {(isSellingLive || allowPurchase) &&
-              !purchased &&
-              withGuaranteeBadge && (
-                <div data-guarantee-image="">
-                  <Image
-                    src="https://res.cloudinary.com/total-typescript/image/upload/v1669928567/money-back-guarantee-badge-16137430586cd8f5ec2a096bb1b1e4cf_o5teov.svg"
-                    width={130}
-                    height={130}
-                    alt="Money Back Guarantee"
-                  />
-                </div>
-              )}
-            {showAllContent && Boolean(modules || features) ? (
-              <div data-header="">
-                <div>
-                  <span>includes</span>
-                </div>
-              </div>
-            ) : null}
-            <div data-main="">
-              {showAllContent &&
-                bonuses &&
-                bonuses.length > 0 &&
-                bonuses[0].expiresAt &&
-                quantity === 1 &&
-                !Boolean(merchantCoupon?.type === 'ppp') && (
-                  <Countdown
-                    date={bonuses[0].expiresAt}
-                    renderer={({days, hours, minutes, seconds, completed}) => {
-                      return completed ? null : (
-                        <>
-                          <div data-limited-bonuses="">
-                            <strong>limited offer</strong>
-                            <ul role="list">
-                              {bonuses.map((bonus) => {
-                                return (
-                                  <li key={bonus.slug}>
-                                    <LimitedBonusItem
-                                      module={bonus as any}
-                                      key={bonus.slug}
-                                    />
-                                  </li>
-                                )
-                              })}
-                              <div data-expires-at="">
-                                {mounted ? (
-                                  <span>
-                                    expires in: {days}d : {hours}h : {minutes}m
-                                    : {seconds}s
-                                  </span>
-                                ) : null}
-                              </div>
-                              <div data-disclaimer="">
-                                Offer available for new purchases only. If
-                                you've already purchased both of the courses
-                                this offer does not apply. If you've purchased 1
-                                of the courses, you'll receive the other.
-                              </div>
-                            </ul>
-                          </div>
-                        </>
-                      )
-                    }}
-                  />
+          {showAllContent && (
```

Comment:
this hides all the footer details

i suspect on Total TypeScript we use CSS to reconfigure the pricing widget for various use cases instead of switches like this...